bugsnag/tornado/__init__.py

```diff
@@ -1,13 +1,43 @@
 import tornado
 from tornado.web import RequestHandler
 from tornado.web import HTTPError
-
+import urllib.parse
 import bugsnag
+import json
 
 
 class BugsnagRequestHandler(RequestHandler):
-    def _handle_request_exception(self, exc):
+    def add_tornado_request_to_notification(self, notification):
+        if not hasattr(self, "request"):
+            return
+
+        try:
+            request_tab = {
+                'method': self.request.method,
+                'path': self.request.uri,
+                'GET': {},
```

Comment:
We should be able to populate this with `request.query`

```suggestion
                'GET': self.request.query,
```